webapp/docs/sage-faqs.mdx

```diff
@@ -36,13 +36,13 @@ sidebar_label: FAQs
 
 - When you purchase Sage, you receive a license key for the number of applications you selected. If you use all your licenses, you can always purchase more at **LINK TO PURCHASE SITE**
 
-**6. Does Sage interface with source code configuration management tools?**
+**6. Can Sage pull my source code from configuration management tools?**
 
 - No, Sage does not interface with any source code configuration management tools. You must extract your source code from your configuration management tool and compress it into a zip file before uploading it to Sage.
 
 **7. Is my data and source code safe when using Sage?**
 
-- Yes, **NEED SOME WORDS HERE ABOUT SECURITY**
+- Yes, we have spent considerable time and effort to ensure your data and source code are well protected. Your information will only be passed through HTTPS secure encryption. And we have followed the protocol set forth in the NIST Special Publication 800-171 for protecting controlled unclassified data as well as addressed any critical and severe vulnerabilities on our containers as provided by MITRE’s Common Vulnerabilities and Exposures (CVE) list.
```

Comment:
I recommend leaving out the word "yes". It is one thing to enumerate the precautions that have been taken, but there will always be some amount of risk.

Reply:
Good note...I have removed and will push with next set of changes. Thanks